storage/blob_test.go

```diff
@@ -798,24 +798,23 @@ func (s *StorageBlobSuite) TestSetBlobProperties(c *chk.C) {
 	c.Check(mPut.ContentLanguage, chk.Equals, props.ContentLanguage)
 }
 
-func (s *StorageBlobSuite) createContainerPermissions(accessType ContainerAccessType,
-	timeout int, leaseID string, ID string, canRead bool,
-	canWrite bool, canDelete bool) ContainerPermissions {
-	perms := ContainerPermissions{}
-	perms.AccessOptions.ContainerAccess = accessType
-	perms.AccessOptions.Timeout = timeout
-	perms.AccessOptions.LeaseID = leaseID
+func (perms *ContainerPermissions) createContainerPermissions(accessType ContainerAccessType,
```

Comment:
I'm struggling with the usage on this one. It seems like this is basically a constructor, but it executes on an existing struct in memory. I suspect this should be updated to:
``` go
func newContainerPermissions(accessType ContainerAccessType, ID string, start, expiry time.Time, canRead, canWrite, canDelete bool) *ContainerPermissions
```